### ISSUE TITLE
Swap blockquotes with code block in finding remainder file

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/freeCodeCamp#49313]
in this file 
https://github.com/freeCodeCamp/freeCodeCamp/blob/bcc9beff1feffe611965ccb95060c42eae8fe864/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/finding-a-remainder-in-javascript.md

We have these quotes, which describe a code example. This breaks the RTL layout, and working around it is weird because quotes should be used to mention previous sentences or previous challenge info.

![image](https://user-images.githubusercontent.com/88248797/218005816-155a465b-8598-4a67-902f-37587b11cfd4.png)

To sort this, we can use code block instead of quotes to display the example.

instead of 
> 5 % 2 = 1 because
> Math.floor(5 / 2) = 2 (Quotient)
> 2 * 2 = 4
> 5 - 4 = 1 (Remainder)

we use

```js
const five = 5;
const two = 2;
const quotient = Math.floor(5 / 2);
console.log(quotient); // logs 2
```

and instead of

> 17 % 2 = 1 (17 is Odd)
> 48 % 2 = 0 (48 is Even)

we use

```js
const oddNumber = 17;
const evenNumber = 48;
```

----

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

**Comment by Vansh-Baghel** [comment]
@Sboonny ok so we have to replace :
This format : `5 % 2 = 1 because
Math.floor(5 / 2) = 2 (Quotient)
2 * 2 = 4
5 - 4 = 1 (Remainder)`
To this format : `const five = 5;
const two = 2;
const quotient = Math.floor(5 / 2);
console.log(quotient); // logs 2` 
Am I Right?

**Comment by Sboonny** [comment]
Yeab, and swap quote format (>)  with codeblock format (\`\`\`)

**Comment by hbar1st** [comment]
I think we should keep the examples that show how the module is working. That is the whole point of this step. (But yes they need to be written in a code block and also need to be reworked to look like code with comments)

**Comment by Vansh-Baghel** [comment]
Affected Page : https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/finding-a-remainder-in-javascript
I got it. The blocks must look more like code rather than maths format. Adding variables would give a better experience of coding to the users.

**Comment by hbar1st** [comment]
This is trickier than expected. I found out that Math.floor is not introduced till a later topic on generating random numbers. We should really remove it from this explanation. 
I would say this explanation needs to focus on modulo and how it works only.
For eg, maybe the step is changed as follows:

============================================
Example:
<code>5%2=1</code>
<code>4%2=0</code>

This is because 5 is an odd number and when divided by 2 it will give a remainder of 1.
While 4 is an even number and when divided by 2 it will give a remainder of 0.

Other examples of modulo operator:

<code>5%3=2</code>
<code>6%3=0</code>
==============================================

**Comment by Vansh-Baghel** [comment]
I seee, good point. Is there a separate question on Math.floor?
Also, according to me, we can give a small intro or example of Math.floor because not everyone would go through each question in a sequence. Math.floor is basic so even a small example would do the job.

**Comment by hbar1st** [comment]
I would not mention Math.floor at all. It is covered in a later step after the learners have learned how to use functions.
Understanding modulo does not require it either as it is a math concept.

**Comment by Nirajn2311** [comment]
Yes you're right @hbar1st . If a topic is covered in a later challenge then we don't mention it in the earlier challenge so as to not confuse the user.

> we can give a small intro or example of Math.floor because not everyone would go through each question in a sequence.

`Math.floor` is a function and the user has not learnt about functions till the challenge being discussed in this issue so this could end up confusing new users as they're more likely to go through the challenges in sequence.

**Comment by Vansh-Baghel** [comment]
So may I start working on it if we have reached the conclusion?

**Comment by hbar1st** [comment]
> So may I start working on it if we have reached the conclusion? 
> 
> 

As you are no longer a first-time contributor(?), then please leave this fix for someone who has not contributed before.

**Comment by Sboonny** [comment]
Before opening this for contribution, can we have a clear example of what the example should be.

We shouldn't use `Math.floor` and `const`, so is this the desired examples?

```js
var five = 5;
var two = 2;
var quotient = five / two;
console.log(quotient); // logs 2
```

```js
const oddNumber = 17;
const evenNumber = 48;
```

BTW, comments inside the markdown `description` aren't translatable as far as I know, I may be wrong tho.

**Comment by Sboonny** [comment]
I just noticed, why is there a mentioning of Quotient in the example? this confused me

**Comment by Vansh-Baghel** [comment]
> > So may I start working on it if we have reached the conclusion?
> 
> As you are no longer a first-time contributor(?), then please leave this fix for someone who has not contributed before.

Oh, got it. Label must be put for first timers.

**Comment by Sboonny** [comment]
Ya, I have removed the label, because I don't want the campers to add changes, then tell them "you know what, we have changed our mind, can you do this instead"

**Comment by Vansh-Baghel** [comment]
> I just noticed, why is there a mentioning of Quotient in the example? this confused me
> 
> Edit: updated the example

Maybe it want to potray that the answer after dividing is the quotient ie the number after dividing 2 other numbers.

**Comment by hbar1st** [comment]
> Example:
> 5%2=1
> 4%2=0
> 
> This is because 5 is an odd number and when divided by 2 it will give a remainder of 1.
> While 4 is an even number and when divided by 2 it will give a remainder of 0.
> 
> Other examples of modulo operator:
> 
> 5%3=2
> 6%3=0

This is what I think it should say.

**Comment by Sboonny** [comment]
> This is what I think it should say. (The numbers and equations should all be in backticks)

Is this accessible to put math equations in backticks? 😮‍💨 I don't want to argue, just use MathJax to place them in mathematical expressions

**Comment by hbar1st** [comment]
I am not sure I understood the question about accessibility for backticks. The curriculum already uses backticks around numbers in the JS course. The equations I am suggesting are almost exactly the same as the ones that are in use in this page currently in terms of the use of the modulo operator and an equal sign.

The idea is to explain how modulo works and how to use it to recognize an even and odd number. While also avoiding bidi issues in the translation (I did not include words in the same line as the equations).

**Comment by Sboonny** [comment]
@bbsmooth, sorry for the ping, but do you know if it is okay to use mathematical expressions inside a code block?

I prefer using MathJax, because they mention taking it seriously, [here are the docs](https://docs.mathjax.org/en/latest/basic/accessibility.html).

Here is markdown file with MathJax in it https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-326-modulo-summations.md

Here is a live URL in the production: https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-326-modulo-summations

--- 

Hopefully, I am worried for nothing, and the experience for screen reader are fine.

**Comment by bbsmooth** [comment]
> okay to use mathematical expressions inside a code block?

I think for simple mathematical expressions like this it is fine. As long as we define the `%` operator (which it appears we do), then a screen reader user will know what that operator represents in the equation.

In my previous job we used Mathjax quite a bit and it is very helpful (especially for complex equations), but it is not perfect and does have some accessibility issues. This has a lot to do with the fact that not all browsers and screen readers support MathML yet. So for these simple equations I would recommend just putting them in a code block over using Mathjax anyway.
